ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapperTest.java

```diff
@@ -250,32 +261,32 @@ public void mapToCCDDefendantWithNullSettlement() {
         Claim claimWithCCJ = SampleClaim.getWithSettlement(null);
 
         //When
-        CCDDefendant ccdDefendant = mapper.to(theirDetails, claimWithCCJ);
+        CCDCollectionElement<CCDDefendant> ccdDefendant = mapper.to(theirDetails, claimWithCCJ);
+        CCDDefendant value = ccdDefendant.getValue();
 
         //Then
-        assertNull(ccdDefendant.getSettlementPartyStatements());
+        assertNull(value.getSettlementPartyStatements());
     }
 
     @Test
     public void mapToCCDDefendantWithSettlements() {
         //Given
         TheirDetails theirDetails = SampleTheirDetails.builder().organisationDetails();
         Claim claimWithSettlement = SampleClaim
-            .getWithSettlement(
-                SampleSettlement.builder().withPartyStatements(
-                    SamplePartyStatement.offerPartyStatement,
-                    SamplePartyStatement.acceptPartyStatement)
-                    .build());
+            .getWithSettlement(SampleSettlement.builder()
+                .withPartyStatements(offerPartyStatement, acceptPartyStatement).build());
+
         final LocalDateTime settlementReachedAt = claimWithSettlement.getSettlementReachedAt();
 
         //When
-        CCDDefendant ccdDefendant = mapper.to(theirDetails, claimWithSettlement);
+        CCDCollectionElement<CCDDefendant> ccdDefendant = mapper.to(theirDetails, claimWithSettlement);
+        CCDDefendant value = ccdDefendant.getValue();
```

Comment:
variable name corrections from value to something proper pls

Reply:
done